### ISSUE TITLE
Dionysus breaks GlowAPI

### ISSUE THREAD

**Issue by MarionetteAccount** [nopjmp/Dionysus#16]
**Describe the bug**
When I use GlowAPI 1.4.6 to make entities glow, nothing happens. using Paper-1618 instead allows it to happen. It is some packet handling change breaking it.
Error I get:
```
method GlowAPI#setGlowing called with ([CraftCreeper] (ArrayList), GREEN (Color), [CraftPlayer{name=Redacted}] (ArrayList)) threw a NoClassDefFoundError: Could not initialize class org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType
```
**To Reproduce**
I do not have easy steps to reproduce sadly as it requires one to write some code with GlowAPI.

**Plugins**
Irrelevant other than GlowAPI

**Version**
Dionysus 1.12.2 2c7a8a8

**Misc**
Any hint in how I could fix this manually is welcome

**Comment by nopjmp** [comment]
@MarionetteAccount are there any other details regarding this like a stacktrace or line number in the GlowAPI so I can track it down?

**Comment by MarionetteAccount** [comment]
Sadly what is provided in the post is all that I got.

**Comment by nopjmp** [comment]
What versions of the PacketAPI are you using?

**Comment by MarionetteAccount** [comment]
PacketListenerApi version 3.7.1-SNAPSHOT

**Comment by nopjmp** [comment]
I modified the packet handler to match vanilla more so this may work now?

**Comment by MarionetteAccount** [comment]
Is there Java 11 version?

**Comment by nopjmp** [comment]
No. I have an idea to switch paradigms for better compatibility. However, I still strongly suggest to switch to a newer version of Minecraft. 1.12.2 is old and we cannot keep it alive forever.
